package.json

```diff
@@ -14,40 +14,24 @@
     "copy:test-assets": "cpx \"./src/tests/assets/**/{*,.*}\" ./lib/tests/assets"
   },
   "dependencies": {
-    "@bentley/backend-itwin-client": "^2.17.2",
-    "@bentley/bentleyjs-core": "^2.17.2",
-    "@bentley/ecschema-metadata": "^2.17.2",
-    "@bentley/electron-manager": "^2.17.2",
-    "@bentley/frontend-authorization-client": "^2.17.2",
-    "@bentley/geometry-core": "^2.17.2",
-    "@bentley/imodelhub-client": "^2.17.2",
-    "@bentley/imodeljs-backend": "^2.17.2",
-    "@bentley/imodeljs-common": "^2.17.2",
-    "@bentley/imodeljs-frontend": "^2.17.2",
-    "@bentley/imodeljs-i18n": "^2.17.2",
-    "@bentley/imodeljs-quantity": "^2.17.2",
-    "@bentley/itwin-client": "^2.17.2",
-    "@bentley/orbitgt-core": "^2.17.2",
-    "@bentley/presentation-common": "^2.17.2",
-    "@bentley/product-settings-client": "^2.17.2",
-    "@bentley/rbac-client": "^2.17.2",
-    "@bentley/telemetry-client": "^2.17.2",
-    "@bentley/ui-abstract": "^2.17.2",
-    "@bentley/webgl-compatibility": "^2.17.2",
-    "electron": "^11.1.0"
+    "@itwin/insights-client": "0.4.0",
+    "@itwin/core-bentley": "^3.6.1",
+    "@itwin/core-backend": "^3.6.1",
+    "@itwin/service-authorization": "^0.7.0",
```

Comment:
@itwin/node-cli-authorization might be a better fit for this. With service-authorization you might need to invite the "service account" into your iTwin for it to be able to do anything.

Reply:
With "node-cli-authorization" package users would need to create a "Desktop/Mobile" application which does not use client secret.